dev/benchmarks/test_apps/stocks/lib/i18n/stock_strings.dart

```diff
@@ -15,6 +15,7 @@ import 'stock_strings_en.dart';
 import 'stock_strings_es.dart';
 
 // ignore_for_file: type=lint
+// dart format on
```

Comment:
Is there any particular reason we want to write this comment to i18n generated files?

Reply:
It depends on the implementation, so no there's no particular reason. So we can remove this line, if `format: true`.
But we still need `// dart format off` if `format: false`

Reply:
I think we should just get rid of `format: false`, it doesn't provide any value at this point.

That is, in this PR, I don't think we should add any `// dart format x` comments, and we should just concentrate on making `format: true` the default, and stop respecting `format: false` entirely.